### PR TITLE
Contact Select: Add `Allow multiple selections` Setting

### PR DESCRIPTION
**Pull request by AlexGStapleton**
This PR will allow users to set the Select field to allow multiple selections. Selections will be listed in the email `with each, selected, item, separated, by a coma`.

### CODE REVIEW

File: widgets/contact/contact.php [hunk 1]
```diff
@@ -192,6 +192,15 @@ function get_widget_form() {
 						)
 					),
 
+					'multiple_select' => array(
+						'type'        => 'checkbox',
+						'label'       => __( 'Allow multiple selections', 'so-widgets-bundle' ),
+						'state_handler' => array(
+							'field_type_{$repeater}[select]' => array( 'show' ),
+							'_else[field_type_{$repeater}]'      => array( 'hide' ),
```
Comment (Misplon) [inline widgets/contact/contact.php:200]: @AlexGStapleton Is this spacing correct? Thanks.

**Comment by Misplon** [comment]
@AlexGStapleton Can you confirm that the notification email to admin arrives normally when selecting all options. Mine seems to be missing the Message and the select data.

**Comment by AlexGStapleton** [comment]
Ah. There was an issue that resulted in forms after the selection not processing correctly. That's now fixed.

**Comment by Misplon** [comment]
Thanks, mate. Please try the layout below and select some of the select items. The email contents aren't being delivered for me.

[so-widgets-bundle.2508-contact-select.zip](https://github.com/siteorigin/so-widgets-bundle/files/9427857/so-widgets-bundle.2508-contact-select.zip)

**Comment by AlexGStapleton** [comment]
It appears you've provided a build rather than a layout. That's not intentional, right? I ask as the build looks to be named in reference to this PR. Regardless, I've spotted that the messages aren't being directly output so I've set them to output. How does that look?

**Comment by Misplon** [comment]
Thanks, mate. Sorry for the wrong upload for the layout.

Are you seeing the semi-colon in the notification email?

Testing1_-_amisplon_gmail_com_-_Gmail

**Comment by AlexGStapleton** [comment]
Is that fields label empty? If I remove the label that's how it appears for me:
Screenshot 2022-08-28 at 01-18-48 MailHog

**Comment by Misplon** [comment]
Quite right!

**Review by Misplon — APPROVED** [review]

**Pull request merged**